daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -602,6 +602,16 @@ createDamlAppTests = testGroup "create-daml-app" [gettingStartedGuideTest | not
           step "Build the new UI"
           callCommandSilent "yarn build"
 
+        -- Run end to end testing for the app.
+        withCurrentDirectory (cdaDir </> "ui") $ do
```

Comment:
We probably need to consider moving the whole GSG stuff to a separate test suite soonish. This test  was already really really slow. Adding pupeteer isn’t going to make it faster.

Reply:
Would the point of splitting it up be to allow running them in parallel? We could move this test to `templates`.

Reply:
Moved the GSG tests to a separate target (in the same directory) in 4fb53662fa39e74a5aa64ac19aa68320efe82931